terraform/india/production/main.tf

```diff
@@ -112,8 +113,51 @@ module "nwp_consumer_ecmwf_live_ecs_task" {
   ]
 }
 
-
 # 3.2
+module "nwp_consumer_gfs_live_ecs_task" {
+  source = "../../modules/services/ecs_task"
```

Comment:
In prod we should pin these to git revisions like the rest of the modules, so nothing gets accidentally changed in prod on module updates

Reply:
good catch